src/js/tools.js

```diff
@@ -219,7 +219,7 @@ function handleShareAction(url) {
 var tag_choices;
 var label_choices;
 
-function showEditDialog(edit_type, edit_id, edit_form_id, edit_title, refresh_row_prefix, argument_string) {
+export function showEditDialog(edit_type, edit_id, edit_form_id, edit_title, refresh_row_prefix, argument_string) {
```

Comment:
another one i didn't see was object edit.
this one doesn't send the event so it's not going to work.

When i add this or event in the function it's missing the dialog buttons
```
jQuery.Deferred exception: dialog_buttons is not defined showEditDialog
```